### PR TITLE
Submit application e2e test

### PR DESCRIPTION
**Pull request by petertgiles**
This branch adds an e2e test to submit an application to a pool.

## Notes
1) Submitting an application is not an idempotent operation - doing it once will prevent you from doing it again.  Therefore, this test runs a fresh db migration and seed before every run.  It's slow but I'm not sure how else to do it.  I opened #4019 to look into it.  I wonder if we should consider a special fast seeder for e2e tests?  Maybe we want to consider squashing our migrations at some point?

1) This branch rejigs the user seeding a bit.  It seeds the admin user, then seeds the pools owned by that user, then seeds the rest of users, including one that has the required skills for the pool.

## Testing
`npx cypress run --headed --spec "cypress/integration/talentsearch/submit-application-workflows.spec.js"`

### CODE REVIEW

**Comment by esizer** [comment]
I'm getting the following error when attempting to run the test. 

```
cy.exec('php ../api/artisan migrate:fresh --seed') failed because the command exited with a non-zero code.

...

SQLSTATE[08006] [7] could not translate host name "postgres" to address: nodename nor servname provided, or not known (SQL: select tablename from pg_catalog.p...
```

**Comment by petertgiles** [comment]
> I'm getting the following error when attempting to run the test.

Shoot, this is another docker vs local issue I'm sure.  You're testing in the php container or maintenance container?

File: frontend/cypress/integration/talentsearch/submit-application-workflows.spec.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    cy.visit("/en/search");
+  });
+
+  it("Submits an application to the CMO pool", () => {
+    cy.login("applicant");
+    cy.visit("/en/browse/pools");
+
+    // Browse pools page
+    cy.wait("@gqlbrowsePoolsQuery");
```
Comment (brindasasi) [inline frontend/cypress/integration/talentsearch/submit-application-workflows.spec.js:29]: I get stuck here. It gets time out.
Submit Workflow Tests -- Submits an application to the CMO pool (failed)

**Review by brindasasi — CHANGES_REQUESTED** [review]
I would think of setting up the data before each run and tear down after the test is done. 
In this case we could delete the application after successful verification. 
We might not need fresh seeding every run.

File: frontend/cypress/integration/talentsearch/submit-application-workflows.spec.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import { aliasMutation, aliasQuery } from "../../support/graphql-test-utils";
+
+describe("Submit Workflow Tests", () => {
+
+  beforeEach(() => {
+    // reseed database to ensure no existing application
+    cy.exec('php ../api/artisan migrate:fresh --seed');
```
[the remark this answers is not in the document]
Comment (brindasasi) [reply frontend/cypress/integration/talentsearch/submit-application-workflows.spec.js:7]: btw , the test works perfectly fine.

**Pull request closed without merging**